source/components/inputs/select/select.ng1.ts

```diff
@@ -96,10 +96,8 @@ export class SelectController extends InputController {
 		this.showOptions = !this.showOptions;
 	}
 
-	close: { (): void } = () => {
-		if (this.showOptions) {
```

Comment:
I was thinking about that after I commented. I can't really think of a reason, so we can probably leave it as-is.